contracts/BinaryOptionMarketManager.sol

```diff
@@ -236,70 +212,62 @@ contract BinaryOptionMarketManager is Owned, Pausable, MixinResolver, IBinaryOpt
     function createMarket(
         bytes32 oracleKey,
         uint strikePrice,
-        bool refundsEnabled,
-        uint[2] calldata times, // [biddingEnd, maturity]
-        uint[2] calldata bids // [longBid, shortBid]
+        uint maturity,
+        uint initialMint // initial sUSD to mint options for
     )
         external
         notPaused
         returns (
             IBinaryOptionMarket // no support for returning BinaryOptionMarket polymorphically given the interface
         )
     {
-        _systemStatus().requireSystemActive();
```

Comment:
marketCreationEnabled and paused flags are enough to control downtimes if ever needed.